octavia-cli/octavia_cli/entrypoint.py

```diff
@@ -67,9 +70,28 @@ def set_context_object(ctx: click.Context, airbyte_url: str, workspace_id: str,
     default=True,
     help="Enable or disable telemetry for product improvement.",
 )
+@click.option(
+    "--header-name",
+    envvar="AIRBYTE_HEADER_NAME",
+    default=None,
+    help="Additional header name to use to call the API, useful when your airbyte instance is secured",
+)
+@click.option(
+    "--header-value",
+    envvar="AIRBYTE_HEADER_VALUE",
+    default=None,
+    help="Additional header value to use to call the API, useful when your airbyte instance is secured",
+)
```

Comment:
Do you think you can come up with an approach that allows to set multiples headers? I would suggest something like:
`octavia --api-header my_header_name:my_headervalue  --api-header my_header_name_2:my_headervalue_2`

And we might also allow to pass a key-value file the CLI can read to get the headers:

`octavia --api-header-file my_api_headers.txt`  

**my_api_headers.txt**:
```
my_header_name=my_header_value
my_header_name_2=my_header_value_2
```

What do you think?

Reply:
I like both ideas.

Reply:
I like both ideas.